docs/faq.md

```diff
@@ -0,0 +1,17 @@
+# FAQ
+
+**Is Shift Available on all Operating Systems?**
+
+Shift is available for all major 64-bit operating systems including Windows and Linux. MacOS compatability testing is currently in progress.
+
+**How many licenses do I need?**
+
+A node-locked license is required for every machine that shift will be running on to build and execute shift workflow graphs.
+We also offer interactive licenses which can be shared amongst machines one at a time.
+Batch Licenses are available for executing workflows only.
+
+**What software is Shift compatible with?**
+
+Shift is currently compatible with versions of Maya, Nuke, Houdini and Blender supporting Python ≥3.7
```

Comment:
Maybe we should leave Blender of at the moment as we barely have any nodes for Blender?

Reply:
Well, shift can be opened in Blender and people can develop nodes using the shift API or prototype using the python script node. I think we should keep it. Any other opinion on this @mromeoinbibo @juliavisserinbibo @craigskerryinbibo @JorgeHernandezibanezinbibo ?

Reply:
I would keep it